dvc/main.py

```diff
@@ -21,6 +22,26 @@
 logger = logging.getLogger("dvc")
 
 
+@contextmanager
+def profile(arg):
+    if arg == -1:
```

Comment:
`arg` is not pretty, but we can now do following:
```
$ dvc list . --cprofile
# prints all stats
$ dvc list . --cprofile dvc.prof
# dumps cprofile results in a dvc.prof file
```

Reply:
@skshetry Worried about it being confusing when you don't place it at the end of the command. How about just `--cprofile-out` ? We could potentially do `--cprofile-sort` in the future too.